### PR TITLE
Use wait node in VSCode for long running computation of children

### PR DESCRIPTION
**Pull request by thurka**
This PR tries to fix the issue, which manifest itself as VSCode error "Element with id xxx is already registered" when expanding nodes in Database view in VSCode. 
It turns out that the error was caused by asynchronous computation of children used netbeans database module and usage of `node.getChildren().getNodes(true)` in LSP server. This implementation causes that VSCode does not see `wait node` which is normally displayed in NetBeans, because `getNodes(true)` waits for all nodes and efectively invalides the asynchronous computation of children for the VSCode. However property change events and other changes for newly created nodes are fired from netbeans code and translated via LSP to VSCode - I believe that this causes mismatch between VSCode and LSP server. 
The fix changes `node.getChildren().getNodes(true)` to `node.getChildren().getNodes()` so that VSCode sees `wait node` and asynchronously computed nodes the same way as are presented in NetBeans. It is also not necessary to wrap `node.getChildren().getNodes()` in `CompletableFuture.supplyAsync()`, since `getNodes()` is designed not to block. Finally `wait node` icon was changed to use `watch` VCSode icon so that the UI of `wait node` match the VSCode LAF.

### CODE REVIEW

**Review by sdedic — APPROVED** [review]
Thanks !!

**Comment by neilcsmith-net** [comment]
@sdedic this cause of failing tests on master?

**Comment by thurka** [comment]
LSP tests are failing after this change. They need to be fixed somehow. I will investigate. However,
as you can [here](https://github.com/apache/netbeans/actions/runs/4510372819) everything was green before merging. It looks like you run additional run after merge with the same result. The real issue is that the LSP tests were not run for this pull request. I am not familiar with the infrastructure, so I cannot tell what went wrong.

**Comment by neilcsmith-net** [comment]
@thurka needs LSP label on opening PR, or locking/unlocking the conversation to trigger a run after the label has been added.  Tried to see if I could trigger after merge.

@sdedic always a good idea to double check the detail of which tests have actually run before merging. :smile:

**Comment by mbien** [comment]
hi @thurka, tests are label activated. The CI run however won't take labeling changes into account *after* it was started, which is an implementation detail of github. Looks like this is what happened here. The tests ran already, after that the label was added without restarting the tests.

there is also a wiki page which explains the issue with PR labeling a bit for reviewers:
https://cwiki.apache.org/confluence/display/NETBEANS/PRs+and+You+-+A+reviewer+Guide

Usually I go around and look out of things like this but was busy recently.

Its not your fault since you are not a committer and can't label anything.

**Comment by thurka** [comment]
You are free to rollback it until the tests are fixed.

**Comment by mbien** [comment]
@thurka it is just a workaround for reviewers to have a way to start a fresh workflow. A regular PR sync does the same but would require more explaining - pressing a button is easier. And this is the only button which doesn't send any emails.

**Comment by neilcsmith-net** [comment]
Thanks @thurka - probably a good idea to revert if no imminent fix as other PR tests failing.  Sorry my explanation was a little brief there - @mbien better!

@mbien @sdedic please merge #5733 if you agree.

**Comment by junichi11** [comment]
@thurka is one of the initial committers: https://cwiki.apache.org/confluence/display/incubator/NetBeansProposal
So, @thurka should be able to add labels himself if he has an Apache id.

**Comment by neilcsmith-net** [comment]
@junichi11 the original proposal list is no longer relevant.  There are various names on there that are not committers.  Please see the Apache Phonebook eg. https://people.apache.org/phonebook.html?unix=netbeans  Anyone not listed there would need voting in as a committer now.

**Comment by mbien** [comment]
@junichi11 @neilcsmith-net well. we do have the option to register accounts as "collaborators". This would provide the labeling perk. This can be done with immediate effect (its just a commit) and would be a lower barrier than committer.

I sent a mail to nb private a few days ago.

**Pull request merged**
